src/utils/report.js

```diff
@@ -57,6 +57,7 @@ export default function ({
 
   const severity = get(result.stylelint, [ "ruleSeverities", ruleName ], "ignore")
 
+  /* istanbul ignore if */
   if (typeof severity === "undefined") {
```

Comment:
Ok, I see. This chunk of code should be deleted then, right? Instead of just ignoring it in code coverage?